Bottom: 460c86bf6e9e259889586ab1090fbff5d9a323a5
Top:    cea7c1ccb2c8d51cf18152528805a86edd456181
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-09-10 17:22:24 +0100

Refresh of crazy-stuff

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index ae391be..fef348d 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1635,6 +1635,7 @@ ghid_draw_everything (BoxTypePtr drawn_area)
         drawn_groups[ngroups++] = group;
       }
     }
+  }
 
   /*
    * first draw all 'invisible' stuff
